### PR TITLE
Add paper-progress to hassio install/update processes

### PR DESCRIPTION
**Pull request by sanyatuning**
## Proposed change
Display `paper-progress` when updating Core or installing/updating an addon.

Supervisor PR: https://github.com/home-assistant/supervisor/pull/1805

Something like this:
progress2

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

Note: You need HA supervisor development env to test this.
```yaml
hassio:
  development_repo: ~/repos/home-assistant/frontend/
```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: hassio/src/components/hassio-progress.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import { SubscribeMixin } from "../../../src/mixins/subscribe-mixin";
+import { HomeAssistant } from "../../../src/types";
+import "@polymer/paper-progress/paper-progress";
```
Comment (bramkragten) [inline hassio/src/components/hassio-progress.ts:13]: Can we use [`mwc-linear-progress`](https://github.com/material-components/material-components-web-components/tree/master/packages/linear-progress)?

Comment (zsarnett) [reply hassio/src/components/hassio-progress.ts:13]: YAS. I actually came to this PR to say this :)

Comment (sanyatuning) [reply hassio/src/components/hassio-progress.ts]: Pls take a look

**Comment by bramkragten** [comment]
Parent PR was closed

**Pull request closed without merging**